### PR TITLE
dev: Set up initial admin user and add migrations to Dockerfile

### PR DESCRIPTION
**Pull request by abbyoung**
## Description 

This PR completes some setup so we can deploy Keystone to our dev environment.

* Removes UI to create initial user
* Creates initial test user in `keystone.ts` config if one does not exist
* Adds prisma migration step to Dockerfile
* Adds rds certs to Dockerfile
* Adds docker compose file for optional Postgres local instance (We may need to add other services in the future so I figured starting a compose file wouldn't hurt).
* Updates README with instructions on running locally

Fixes sc-202

## Review Notes

For the rds certs, I mirrored what we did in the portal-client repo. I did have to change the code from `apt-get` to `apk`, so let me know if anything looks off, @esacteksab.

### CODE REVIEW

File: .node-version [hunk 1]
```diff
@@ -1 +1 @@
-14.18.3
\ No newline at end of file
+14.19.0
```
Comment (abbyoung) [inline .node-version:1]: @suzubara My machine was giving me problems and I had to update to this version to get it working. Is this the version we should be on, or should I change back + hit my computer with a hammer?

Comment (abbyoung) [reply .node-version]: jk i fixed it!

File: keystone.ts [hunk 2]
```diff
@@ -17,19 +17,34 @@ import { withAuth, session } from './src/lib/auth'
...
   // Using the config function helps typescript guide you to the available options.
   config({
-    // the db sets the database provider - we're using sqlite for the fastest startup experience
+    lists,
     db: {
       provider: 'postgresql',
       url: process.env.DATABASE_URL || '',
       enableLogging: true,
-      useMigrations: false, // TODO - set to true after deploying
+      useMigrations: false,
```
Comment (suzubara) [inline keystone.ts:25]: Can we leave the TODO comment here about switching this to `true` eventually?

File: migrations/20220210192334_/migration.sql [hunk 3]
```diff
@@ -0,0 +1,56 @@
+-- CreateTable
```
Comment (suzubara) [inline migrations/20220210192334_/migration.sql:1]: Should this file be here if `useMigrations` is false?

File: Dockerfile [hunk 4]
```diff
@@ -15,7 +15,9 @@ COPY . .
 
 ENV NODE_ENV development
 
-RUN yarn install --frozen-lockfile && yarn build && yarn cache clean
+RUN yarn install --frozen-lockfile \
+    # && yarn keystone prisma migrate dev \
```
Comment (suzubara) [inline Dockerfile:19]: can we remove this line if it's not needed

File: migrations/20220210192334_/migration.sql (hunk 3 above)
Comment (abbyoung) [reply migrations/20220210192334_/migration.sql:1]: yes! our migrations folder is checked/any outstanding ones are applied during the prisma migration we run in the Dockerfile. that's what sets up the database correctly for our test user to be created in `onConnect`

if anything, i'm wondering if `useMigrations` should be true?

**Review by suzubara — APPROVED** [review]
beautiful! looks good to me but I think we should wait for confirmation from @esacteksab before merging

File: README.md [hunk 5]
```diff
@@ -2,13 +2,35 @@
...
+
+Set these variables for Keystone in a `.envrc.local` file.
+
+- `SESSION_SECRET` (must be a string at least 32 chars)
+- `DATABASE_URL` (URL to a running Postgres instance)
+- `TEST_USERNAME` (string)
+- `TEST_EMAIL` (string - used to log in to Admin UI)
+- `TEST_PASSWORD` (must be a string at least 8 chars - used to log in to Admin UI)
+
+Environment variables for Postgres are set in the `docker-compose.yml` file. The initial database user created by Postgres will be `postgres` with the password set in the compose file. The database created will be named `keystone`, also set in the compose file.
```
[the remark this answers is not in the document]
Comment (esacteksab) [reply README.md]: I was using the same username we used everywhere else `iamgroot` but I can easily change it to `keystone`

**Comment by esacteksab** [comment]
> Yay this is awesome & works as expected! I did have to change the node version to 14.19.0 in order to build because it looks like the docker image for `node:14-alpine3.15` recently updated to that version. I don't think updating our version is an issue.

If you don't want 14.19.0 yet, you can pin to 14.18.3 with this image tag https://hub.docker.com/layers/node/library/node/14.18.3-alpine3.15/images/sha256-171467cfe1394f022a89cf099931e1dd1cd60702e3dbbc16138db471d3d67043?context=explore

File: README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (abbyoung) [reply README.md]: @esacteksab these variables are just for running the postgres container locally, so yours should be fine

**Pull request merged**